docs/dashboards/data-handling.md

```diff
@@ -87,3 +123,35 @@ supports multiple tables and statuses simultaneously. Each Board has a DataCurso
 that is accessible via the `Board.dataCursor` property.
 
 <iframe style="width: 100%; height: 450px; border: none;" src="https://www.highcharts.com/samples/embed/dashboards/sync/datacursor-sync" allow="fullscreen"></iframe>
+
+
+## GoogleSheetsConnector Tutorial
```

Comment:
Could you please move this part to `docs/dashboards/data-pool-and-connectors.md`